format_print.c

```diff
@@ -379,7 +379,7 @@ static struct expr *format_parse_cond(const char* format, int size)
 static uchar format_skip_cond_expr(const char *format, int *s)
 {
 	uchar r = 0;
```

Comment:
What the hell is this variable anyway. It's not even used anywhere. This code seems seriously broken.